src/services/topicTemplate/model.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const mongoose = require('mongoose');
```

Comment:
Codacy found an issue: [Expected 1 empty line after require statement not followed by another require.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045819)